view_tasker_project_summary.py

```
@@ -0,0 +1,201 @@
+            padded = []
+            for i, cell in enumerate(row):
+                if i < len(col_widths):
+                    padded.append(cell.ljust(col_widths[i]))
+                else:
+                    padded.append(cell)
+            print('  '.join(padded))
+
+    except FileNotFoundError:
+        print(f"Error: File not found: {file_path}", file=sys.stderr)
+        sys.exit(1)
+    except Exception as e:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)